### ISSUE TITLE
Using error variable name while using destructuring object as paramater with default value

### ISSUE THREAD

**Issue by NextSeason** [rollup/rollup#1008]
main.js

``` javascript
import { ajax } from './http.js';
class Test {
    constructor() {
        this.ajax = false;
    }
    volumn( { ajax = true } ) {
        this.ajax = ajax;   
    }
}

export default Test;
```

http.js

``` javascript
export var ajax = ( init ) => {
    console.log( init );
};
```

output.js

``` javascript
define(function () { 'use strict';

class Test {
    constructor() {
        this.ajax = false;
    }
    volumn( { ajax = true } ) {
        this.ajax = ajax$$1;    
    }
}

return Test;

});
```

I think it should not use 'ajax$$1' in method volumn.

**Comment by Victorystick** [comment]
Reproduction of the issue [here](http://rollupjs.org/?version=0.36.1&shareable=JTdCJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmFtZCUyMiUyQyUyMm1vZHVsZU5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlMkMlMjJtb2R1bGVJZCUyMiUzQSUyMiUyMiU3RCUyQyUyMm1vZHVsZXMlMjIlM0ElNUIlN0IlMjJuYW1lJTIyJTNBJTIybWFpbi5qcyUyMiUyQyUyMmNvZGUlMjIlM0ElMjJpbXBvcnQlMjAlN0IlMjBhamF4JTIwJTdEJTIwZnJvbSUyMCcuJTJGaHR0cC5qcyclM0IlNUNuY2xhc3MlMjBUZXN0JTIwJTdCJTVDbiUyMCUyMCUyMCUyMGNvbnN0cnVjdG9yKCklMjAlN0IlNUNuJTIwJTIwJTIwJTIwJTIwJTIwJTIwJTIwdGhpcy5hamF4JTIwJTNEJTIwZmFsc2UlM0IlNUNuJTIwJTIwJTIwJTIwJTdEJTVDbiUyMCUyMCUyMCUyMHZvbHVtbiglMjAlN0IlMjBhamF4JTIwJTNEJTIwdHJ1ZSUyMCU3RCUyMCklMjAlN0IlNUNuJTIwJTIwJTIwJTIwJTIwJTIwJTIwJTIwdGhpcy5hamF4JTIwJTNEJTIwYWpheCUzQiUyMCUyMCUyMCU1Q24lMjAlMjAlMjAlMjAlN0QlNUNuJTdEJTVDbiU1Q25leHBvcnQlMjBkZWZhdWx0JTIwVGVzdCUzQiUyMiU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJodHRwLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMHZhciUyMGFqYXglMjAlM0QlMjAoJTIwaW5pdCUyMCklMjAlM0QlM0UlMjAlN0IlNUNuJTIwJTIwJTIwJTIwY29uc29sZS5sb2coJTIwaW5pdCUyMCklM0IlNUNuJTdEJTNCJTIyJTdEJTVEJTdE).

**Comment by Rich-Harris** [comment]
Fixed in 0.38 – thanks
